subjects/CPM-80/CB80.c

```diff
@@ -236,7 +236,7 @@ byte fn05CE(Eq_n bc)
 		fn056B(SEQ(b_n, *globals->t166D), out bc_n);
 		globals->t166D = (word16) globals->t166D + 0x01;
 	}
-	return (byte) bc_n;
+	return SLICE(bc_n, byte, 8);
```

Comment:
This is probably an improvement, as it shows that previously, the cast was returning the `C` part of the  `BC` register pair, but here it's returning `B`, the high byte.